src/otx/algo/classification/torchvision_model.py

```diff
@@ -0,0 +1,250 @@
+        return TVModelWithLossComputation(
+            backbone=self.backbone,
+            num_classes=self.num_classes,
+            loss=self.loss,
+        )
+
+    def _customize_inputs(self, inputs: MulticlassClsBatchDataEntity) -> dict[str, Any]:
+        if isinstance(inputs.images, list):
+            images = tv_tensors.wrap(torch.stack(inputs.images, dim=0).to(dtype=torch.float32), like=inputs.images[0])
+        else:
+            images = inputs.images.to(dtype=torch.float32)
+        return {
+            "images": images,
+            "labels": torch.cat(inputs.labels, dim=0),
+            "mode": "loss" if self.training else "predict",
+        }
+
+    def _customize_outputs(
+        self,
+        outputs: Any,  # noqa: ANN401
+        inputs: MulticlassClsBatchDataEntity,
+    ) -> MulticlassClsBatchPredEntity | MulticlassClsBatchPredEntityWithXAI | OTXBatchLossEntity:
+        if self.training:
+            return OTXBatchLossEntity(loss=outputs)
+
+        # To list, batch-wise
+        logits = outputs if isinstance(outputs, torch.Tensor) else outputs["logits"]
+        scores = torch.unbind(logits, 0)
+        preds = logits.argmax(-1, keepdim=True).unbind(0)
+
+        if self.explain_mode:
+            if not isinstance(outputs, dict) or "saliency_map" not in outputs:
+                msg = "No saliency maps in the model output."
+                raise ValueError(msg)
+
+            saliency_maps = outputs["saliency_map"].detach().cpu().numpy()
+
+            return MulticlassClsBatchPredEntityWithXAI(
+                batch_size=len(preds),
+                images=inputs.images,
+                imgs_info=inputs.imgs_info,
+                scores=scores,
+                labels=preds,
+                saliency_maps=list(saliency_maps),
+                feature_vectors=[],
+            )
+
+        return MulticlassClsBatchPredEntity(
+            batch_size=inputs.batch_size,
+            images=inputs.images,
+            imgs_info=inputs.imgs_info,
+            scores=scores,
+            labels=preds,
+        )
+
+    @property
+    def _export_parameters(self) -> dict[str, Any]:
+        """Defines parameters required to export a particular model implementation."""
+        export_params: dict[str, Any] = {}
+        export_params["input_size"] = (1, 3, 224, 224)
+        export_params["output_names"] = ["logits", "saliency_map"] if self.explain_mode else None
+        export_params["resize_mode"] = "standard"
+        export_params["pad_value"] = 0
+        export_params["swap_rgb"] = False
+        export_params["via_onnx"] = False
+        export_params["onnx_export_configuration"] = None
+        export_params["mean"] = [0.485, 0.456, 0.406]
+        export_params["std"] = [0.229, 0.224, 0.225]
+
+        parameters = super()._export_parameters
+        parameters.update(export_params)
+        return parameters
+
+    @staticmethod
+    def _forward_explain_image_classifier(
```

Comment:
`image_classifier` is basically lowered name of the class in mmpretrain, for which this forward replaces the original forward. For the torchvision I would propose for example:

```suggestion
    def _forward_explain_torch_vision_model(
```

Reply:
But if I rename the function, don't I have to override `forward_explain` again?
Also, if `_forward_explain_image_classifier` is a function that depends on mmpretrain, shouldn't we move it to an MM-related class instead of `ExplainableOTXClsModel`?